### PR TITLE
Use source CSV file name as ID for samples & don’t use samples with missing composition data

### PR DESCRIPTION
**Pull request by chhoumann**

### CODE REVIEW

File: baseline/PLS_SM/full_flow.py [hunk 1]
```diff
@@ -376,40 +383,42 @@ def get_models(experiment_id: str) -> Dict[str, Dict[str, PLSRegression]]:
...
+
+            pred = predict_composition_with_blending(
+                oxide, X_test_n1, X_test_n3, models, optimized_blending_ranges
+            )
+
+            # save
+            pred_df = pd.DataFrame(pred, index=Y.index)
+            pred_df.to_csv(predictions_save_path / f"{oxide}.csv")
+
+            # ???
```
Comment (Pattrigue) [inline baseline/PLS_SM/full_flow.py:400]: ???

File: baseline/train_test_split.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def get_composition_for_sample(cd, sample_name):
 if __name__ == "__main__":
     save_path = Path("train_test_split.csv")
 
-    if save_path.exists():
+    if save_path.exists() and False:
```
Comment (Pattrigue) [inline baseline/train_test_split.py:41]: and False?

**Comment by chhoumann** [comment]
Aware of `root_mean_squared_error` but we use 1.3.x:
image

**Review by Pattrigue — APPROVED** [review]
Untitled

**Pull request merged**